manager/orchestrator/task.go

```diff
@@ -50,7 +50,7 @@ func NewTask(cluster *api.Cluster, service *api.Service, slot uint64, nodeID str
 	}
 
 	// Assign name based on task name schema
-	name := store.TaskName(&task)
+	name := naming.Task(&task)
 	task.Annotations = api.Annotations{Name: name}
```

Comment:
You can split it into a separate branch if it's cleaner, but it's literally here ^^^

Since you're stopping by it would be too bad not to change it